scripts/tools/k3d.sh

```diff
@@ -0,0 +1,16 @@
+# shellcheck shell=bash
```

Comment:
What is this for ? It looks incomplete, and has typos.

Reply:
https://github.com/koalaman/shellcheck/blob/master/shellcheck.1.md#environment-variables

Reply:
Actually `this` refered to the entire script, not the particular line.